.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: RPC Tests
+
+on:
+  pull_request:
```

Comment:
see [this doc](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions)